.github/workflows/check-new-releases.yaml

```diff
@@ -0,0 +1,335 @@
+name: Check new builds and release them
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+        description: The name of the product to be released.
+      lp-project:
+        required: true
+        type: string
+        description: The LP build project for the product to be released.
```

Comment:
We probably need to rename this one to `lp-build-project` and add an additional project for the releases, i.e: `lp-release-project` (which should default to `{product.name}-releases`)